setup.py

```diff
@@ -21,7 +21,7 @@
     extras_require={
         'call': ['twilio>=6.0'],
         'sms': ['twilio>=6.0'],
-        'webauthn': ['webauthn>=1.6.0,<1.99', 'pydantic>=1.9.0,<1.99'],
+        'webauthn': ['webauthn>=1.6.0,<1.99'],
```

Comment:
shall we follow @Natureshadow 's suggestion "partially" and relax the dependency instead of dropping it?

It's the official path for the symbol until at least <3.0.

Re-importing from `webuthn.entrails.that.might.change` seems mildly more dangerous even if it lets us drop the direct dependency on pydantic.

Reply:
the difference between pydantic and other py_webauthn dependencies is that django-two-factor explicitly depends on ValidationError.